apps/docs-app/docs/features/deployment/providers.md

```diff
@@ -90,9 +90,9 @@ export default defineConfig(({ mode }) => ({
 
 When using Nx and reusing the build cache on the Vercel build platform, there is a possibility that the cache is reused if you have built it locally. This can lead to the output being placed in the wrong location. To resolve this issue, you can use the preset in the `vite.config.ts` file as a workaround.
 
-## Cloudflare Pages and Workers
+## Cloudflare Pages and Functions
```

Comment:
Could you please drop "functions"? It's not necessary in this context since the build output actually uses Workers rather than the higher level Functions it's a bit misleading.

```suggestion
## Cloudflare Pages
```